src/coreComponents/physicsSolvers/solidMechanics/kernels/ExplicitSmallStrain.hpp

```diff
@@ -203,7 +203,7 @@ class ExplicitSmallStrain : public finiteElement::KernelBase< SUBREGION_TYPE,
   arrayView2d< real64, nodes::ACCELERATION_USD > const m_acc;
 
   /// The time increment for this time integration step.
-  real64 const m_dt;
+  real64 const m_dt; ///TODO: Consider moving to finite element kernel base?
```

Comment:
yeah, I think you could probably just move it to the base class. At the end of the day your constitutive model will work with any time integration option.